content/notifications/architecture/sms/_index.en.md

```diff
@@ -5,6 +5,15 @@ tags: [architecture, solution, notifications, sms]
 weight: 2
 toc: false
 ---
+## API
+
+### Public API
+### Altinn Notifications Sms API
+The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports. 
```

Comment:
```suggestion
The Altinn Sms Notifications API is an HTTP-based RESTful API that provides an endpoint for receiving delivery reports from the SMS provider. 
```

Reply:
Could we simplify it even further? 
All details are not required, we could add the suggestion below, and then move the information about link pushing to the integrations section where we cover link? 

### Public API 
 - [DeliveryReportController](https://github.com/Altinn/altinn-notifications-sms/blob/main/src/Altinn.Notifications.Sms/Controllers/DeliveryReportController.cs)
  Endpoint receiving delivery reports in XML-format from SMS provider. The controller is protected with [basic authentication](https://github.com/Altinn/altinn-notifications-sms/blob/main/src/Altinn.Notifications.Sms/Configuration/BasicAuthenticationHandler.cs).